indico/queries/model_groups.py

```diff
@@ -270,19 +272,39 @@ def process_response(self, response):
 
 class _ModelGroupPredict(GraphQLRequest):
     query = """
-        mutation ModelGroupPredict($modelId: Int!, $data: [String], $predictOptions: JSONString) {
-            modelPredict(modelId: $modelId, data: $data, predictOptions: $predictOptions) {
+        mutation ModelGroupPredict(<QUERY_ARGS>) {
+            modelPredict(<MODEL_PREDICT_ARGS>) {
                 jobId
             }
         }
         """
 
+    query_args = {
+        "modelId": "Int!",
+        "data": "[String]",
+        "predictOptions": "JSONString"
+    }
+
+    def _args_strings(self, **kwargs):
+        args = [k for k in self.query_args.keys() if kwargs.get(cc_to_snake(k))]
```

Comment:
Generally think we should prefer loud errors to silent failures but this is internal so less critical